recipes-wigwag/maestro/maestro/rpi3/devicedb.template.conf

```diff
@@ -182,8 +182,8 @@ cloud:
     # historyID: "*.wigwag.io"
     # historyHost and historyPort may be ommitted. In this case historyHost and
     # historyPort are set to the normal cloud host and port
-    historyURI: {{ARCH_GW_SERVICES_URL}}/relay-history/history
-    alertsURI: {{ARCH_GW_SERVICES_URL}}/relay-alerts/alerts
+    historyURI: http://gateways.local:8080/relay-history/history
```

Comment:
@psppalash Can we create a var_defs in maestro and define the fog-proxy local address - "http://gateways.local:8080" there and use mustache in the template files?

Reply:
Will do that